src/features/config/form/addresses.ts

```diff
@@ -118,16 +99,16 @@ export const defaultAddressConfiguration: IAddressConfiguration[] = [
       {
         config: AddressSubsections.PRIMARY_ADDRESS_SUBSECTION,
         label: formMessageDescriptors.primaryAddress,
-        conditionalCase: `${MOTHER_DETAILS_DONT_EXIST}`
+        conditionalCase: MOTHER_DETAILS_DONT_EXIST
       },
       {
         config: AddressCases.PRIMARY_ADDRESS,
-        conditionalCase: `${MOTHER_DETAILS_DONT_EXIST}`
+        conditionalCase: MOTHER_DETAILS_DONT_EXIST
       },
       {
         config: AddressSubsections.SECONDARY_ADDRESS_SUBSECTION,
         label: formMessageDescriptors.secondaryAddress,
-        conditionalCase: `${MOTHER_DETAILS_DONT_EXIST} || ${secondaryAddressesDisabled}`
```

Comment:
This change might be unwanted. Because the previous value looks to be the correct one.